cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -198,12 +197,16 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
             } catch (e: DownloadException) {
                 e.showStackTrace()
 
-                log.error { "Could not download '${pkg.id.toCoordinates()}': ${e.collectMessagesAsString()}" }
+                val errorMessage = "Could not download '${pkg.id.toCoordinates()}': ${e.collectMessagesAsString()}"
+                errorMessages += errorMessage
 
-                error = true
+                log.error { errorMessage }
             }
         }
 
-        if (error) throw UsageError("An error occurred.", statusCode = 2)
+        if (errorMessages.isNotEmpty()) {
+            log.error { "Error Summary: \n\n${errorMessages.joinToString("\n\n")}" }
```

Comment:
Nit: As you add linebreaks anyway, the space after "Summary: " is not needed.

Reply:
Done.